### ISSUE TITLE
Transaction List does not appear in Clams interface

### ISSUE THREAD

**Issue by farscapian** [clams-tech/browser-app#70]
I have some odd functionality happening. After connecting Clams to my CLightning node (I'm running BTCPay Server), I don't see any Transaction list, which i am told is supposed to appear at the bottom. I have tried multiple browsers (brave/firefox/chrome running on Graphene) and combination of desktop (running Ubuntu brave/firefox/chrome) setups, so it appears to not be an issue with browser compatibility. Each new browser session was issued a unique rune according to the instructions under https://docs.clams.tech/authentication/#lightning-payments, so it doesn't appear to be permissions related.

When I visit the /payments route, I get an empty transaction list with an error message of "Not a proper lightning payment request" message. Hoping someone can help me troubleshoot and to the bottom of this issue.

**Comment by aaronbarnardsound** [comment]
Hey @farscapian it seems like you have a `pay` entry in your CoreLN db that has a `bolt11` string value that does not start with `ln` and that is causing the bolt11 decoding library we use to throw an error. I have added better error handling in #73 so it should no longer fail. If you could test it out on the [staging app](https://staging.app.clams.tech) and also report back what the bolt11 value that is logged to the console is, that would be super helpful.

**Comment by farscapian** [comment]
I rrunning the staging version now and I do now see additional transaction options listed at the bottom of the app showing "Received xxx today at 13:00. However, when I visit the /payments route nothing appears. When I clock the vConsole, the following uncaught message appears

Uncaught (in promise)  {"name": "Error", "message": "Cannot have duplicate keys in a keyed each", "stack": "Error: Cannot have duplicate keys in a keyed each\n    at Ue (https://staging.app.clams.tech/_app/immutable/chunks/index-266e4e80.js:4:6427)\n    at Array.St (https://staging.app.clams.tech/_app/immutable/components/pages/payments/_page.svelte-6c0b4eb4.js:33:173)\n    at Array.Et (https://staging.app.clams.tech/_app/immutable/components/pages/payments/_page.svelte-6c0b4eb4.js:33:2770)\n    at fe (https://staging.app.clams.tech/_app/immutable/chunks/index-266e4e80.js:1:887)\n    at b (https://staging.app.clams.tech/_app/immutable/chunks/Slide-6677c26c.js:1:874)\n    at tn (https://staging.app.clams.tech/_app/immutable/chunks/index-266e4e80.js:4:7595)\n    at new ee (https://staging.app.clams.tech/_app/immutable/chunks/Slide-6677c26c.js:1:2498)\n    at He (https://staging.app.clams.tech/_app/immutable/components/pages/payments/_page.svelte-6c0b4eb4.js:33:4412)\n    at tn (https://staging.app.clams.tech/_app/immutable/chunks/index-266e4e80.js:4:7595)\n    at new xt (https://staging.app.clams.tech/_app/immutable/components/pages/payments/_page.svelte-6c0b4eb4.js:33:6588)"}

**Comment by aaronbarnardsound** [comment]
Deployed another fix to staging with #74, try again when you get a minute mate and let me know what you see :)

**Comment by farscapian** [comment]
This error appears on the /payments route:  ![image](https://user-images.githubusercontent.com/23437177/210395368-99928807-2fb2-48c3-be91-5591fdf6cd00.png)

under the vConsole of the /payments route, I get the following:
Uncaught (in promise)  {"name": "Error", "message": "Cannot have duplicate keys in a keyed each", "stack": "Ue@https://staging.app.clams.tech/_app/immutable/chunks/index-266e4e80.js:4:6427\nSt@https://staging.app.clams.tech/_app/immutable/components/pages/payments/_page.svelte-1c89714c.js:33:175\nEt@https://staging.app.clams.tech/_app/immutable/components/pages/payments/_page.svelte-1c89714c.js:33:2770\nfe@https://staging.app.clams.tech/_app/immutable/chunks/index-266e4e80.js:1:887\nb@https://staging.app.clams.tech/_app/immutable/chunks/Slide-6677c26c.js:1:875\ntn@https://staging.app.clams.tech/_app/immutable/chunks/index-266e4e80.js:4:7595\nee@https://staging.app.clams.tech/_app/immutable/chunks/Slide-6677c26c.js:1:2498\nHe@https://staging.app.clams.tech/_app/immutable/components/pages/payments/_page.svelte-1c89714c.js:33:4412\ntn@https://staging.app.clams.tech/_app/immutable/chunks/index-266e4e80.js:4:7595\nxt@https://staging.app.clams.tech/_app/immutable/components/pages/payments/_page.svelte-1c89714c.js:33:6588\np@https://staging.app.clams.tech/_app/immutable/start-a0c7cd2c.js:1:6384\n_e@https://staging.app.clams.tech/_app/immutable/chunks/index-266e4e80.js:1:1250\np@https://staging.app.clams.tech/_app/immutable/components/pages/_layout.svelte-1e5906c8.js:28:4299\np@https://staging.app.clams.tech/_app/immutable/components/pages/_layout.svelte-1e5906c8.js:28:5958\nVt@https://staging.app.clams.tech/_app/immutable/chunks/index-266e4e80.js:4:2639\nlt@https://staging.app.clams.tech/_app/immutable/chunks/index-266e4e80.js:4:2317\n"}

**Comment by aaronbarnardsound** [comment]
Ah that error notification explains #71. I realised that we do not have the rune recipe documented properly to include the `waitanyinvoice` method, so I have updated that. Try recreating a rune with the [updated command here](https://docs.clams.tech/authentication/#lightning-payments).

The duplicate keys error means that you must have two payments that have the same `payment_hash` which I did not think was possible. In any case I have merged an update with #77 that should fix it, so try again on the [staging app](https://staging.app.clams.tech) when you get a min.
